Assets/Utilities/MyUtils.cs

```diff
@@ -0,0 +1,24 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+/// <summary>
+/// Utility functions.
+/// </summary>
+public static class MyUtils
+{
+    /// <summary>
+    /// Project the provided <c>objTransform.forward</c> vector onto the World-space X-Z plane.
```

Comment:
Having Y be up is grinding my gears.

Reply:
oh I hate it. _ALSO_ Unity is _left-handed_ which has wasted _so many_ of my hours